utils/configuration_test.go

```diff
@@ -335,3 +335,30 @@ func TestParseTLSWithEnvironmentVariables(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, expected, *tlsOpts)
 }
+
+func TestParseViperWithInvalidFile(t *testing.T) {
+	v := viper.New()
+	SetupViper(v, envPrefix)
+
+	err := ParseViper(v, "Chronicle_Of_Dark_Secrets.json")
+	assert.Error(t, err)
+	assert.Contains(t, err.Error(), "Could not read config")
+}
+
+func TestParseViperWithValidFile(t *testing.T) {
+	file, err := os.Create("/tmp/Chronicle_Of_Dark_Secrets.json")
+	assert.NoError(t, err)
+	defer os.Remove(file.Name())
+
+	file.WriteString(`{"logging": {"level": "debug"}}`)
+
+	v := viper.New()
+	SetupViper(v, envPrefix)
+
+	err = ParseViper(v, "/tmp/Chronicle_Of_Dark_Secrets.json")
+	assert.NoError(t, err)
+
+	if lvl := v.GetString("logging.level"); lvl != "debug" {
+		t.Fatalf("Expect debug, got %v", lvl)
```

Comment:
Non-blocking, since we don't have a coding standard for tests yet:  we are trying to convert instances of `t.Fatalf` to use `assert` now.  :)  So this would probably be something like `assert.Equal(t, "debug", v.GetString("logging.level"))`.